### PR TITLE
fix: de-emphasize copy to clipboard button

### LINKED ISSUE #6244: `Copy to Clipboard` button is too prominent
A tiny design nit is that the `Copy to Clipboard` button is very prominent in the smaller coding snippets, which I find distracting from the content. This should be a de-emphasized control in my opinion.

image

Compare to the homepage which uses a de-emphasized version

image

_Originally posted by @bmuenzenmeyer in https://github.com/nodejs/nodejs.org/issues/6215#issuecomment-1890272029_

### PR DESCRIPTION
**Pull request by bmuenzenmeyer**
## Description

De-emphasizes the copy to clipboard button. also removed the homepage customization. yes, this is a deviation of the figma- but now met with real content, especially repeated examples are distracting. this is a tertiary act for the site, it should not have primary/action color

## Validation

same homepage

image

less busy examples within content

| Old | New |
| - | - |
| image  |image |

## Related Issues

closes #6244

### Check List

- [x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [x] I have run `npx turbo lint` to ensure the code follows the style guide. And run `npx turbo lint:fix` to fix the style errors if necessary.
- [x] I have run `npx turbo format` to ensure the code follows the style guide.
- [x] I have run `npx turbo test` to check if all tests are passing.
- [x] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

**Comment by ovflowd** [comment]
@bmuenzenmeyer does it make sense to keep the green colour on light theme? Like keeping the gray on dark theme and green on light theme? I have the feeling that within the light theme the gray button is more emphasized than the green one.

**Review by ovflowd — APPROVED** [review]

**Comment by bmuenzenmeyer** [comment]
I thought about that but the codebox is always dark, so this still seems appropriate to me.

Open to more thoughts, or iterating on this for a bit.

**Review by AugustinMauroy — APPROVED** [review]
LGTM

I think that on `en/learn/getting-started/introduction-to-nodejs` the code snippet that displays functions should be inline code.

**Review by shanpriyan — APPROVED** [review]
LGTM!

**Review by aymen94 — APPROVED** [review]

File: components/Common/CodeBox/index.module.css [hunk 1]
```diff
@@ -70,8 +70,15 @@
...
+        border-none
+        bg-neutral-900
         px-3
-        py-1.5;
+        py-1.5
+        font-medium;
+
+      &:hover {
+        @apply bg-neutral-800;
+      }
```
Comment (canerakdas) [inline components/Common/CodeBox/index.module.css]: Wouldn't it be reusable to add a`neutral` kind to the`Button` component instead of overwriting the styles here?

**Review by canerakdas — APPROVED** [review]

File: components/Common/CodeBox/index.module.css (hunk 1 above)
Comment (ovflowd) [reply components/Common/CodeBox/index.module.css]: It does make sense indeed of creating another Button variant, this one could be useful in some places!

Comment (ovflowd) [reply components/Common/CodeBox/index.module.css]: cc @bmuenzenmeyer

File: components/Common/CodeBox/index.module.css
Comment (bmuenzenmeyer) [inline components/Common/CodeBox/index.module.css:1]: this change and the removal in layouts.module.css removes the customization that gives use the neutral color, but retains the customization that allows for both a smaller profile AND also one that accomodates an icon

we could explore adding both to the button component, but it's starting to scope creep IMO

**Review by bmuenzenmeyer — COMMENTED** [review]
added a neutral button `kind` - alphabetically first across all diffs

File: components/Common/CodeBox/index.module.css (hunk 1 above)
Comment (bmuenzenmeyer) [reply components/Common/CodeBox/index.module.css]: addressed via https://github.com/nodejs/nodejs.org/pull/6246/commits/1a8280e1c6468c413623d3256d5c3a2251239175

File: components/Common/CodeBox/index.module.css
Comment (ovflowd) [reply components/Common/CodeBox/index.module.css:1]: > this change and the removal in layouts.module.css removes the customization that gives use the neutral color, but retains the customization that allows for both a smaller profile AND also one that accomodates an icon

wdym?

**Review by ovflowd — APPROVED** [review]

File: components/Common/CodeBox/index.module.css
Comment (bmuenzenmeyer) [reply components/Common/CodeBox/index.module.css:1]: i mean that codebox is the only button that has an icon, and as such we introduced a customization to the button to make it look nice. without these values here the icon would break onto a new line vs the text

image

my comment was that in some design systems, a button and icon is a standard use case that would promote these flexbox styles into the `Button` itself - but I don't think we should do that until we have to

Comment (ovflowd) [reply components/Common/CodeBox/index.module.css:1]: > i mean that codebox is the only button that has an icon, and as such we introduced a customization to the button to make it look nice. without these values here the icon would break onto a new line vs the text

Right I think @canerakdas is fixing that on another PR, that the Button should be flex by default. I wonder if he fixed that already?

Comment (bmuenzenmeyer) [reply components/Common/CodeBox/index.module.css:1]: @canerakdas can you review the above? I don't want us to head into a conflict - but glancing at the PRs I didn't notice anything. I'd like to merge this if we can.

Comment (canerakdas) [reply components/Common/CodeBox/index.module.css:1]: @ovflowd Not yet, I'm planning to create another PR for this since many places are affected

**Comment by ovflowd** [comment]
@bmuenzenmeyer apparently the merge queue is failing. You might want to check why 👀

**Comment by ovflowd** [comment]
@bmuenzenmeyer I guess rebase is needed!

**Comment by bmuenzenmeyer** [comment]
Yes I saw that, just too busy today. 4 sick people here and lice, lol.

Standby

**Comment by bmuenzenmeyer** [comment]
@ovflowd this lint error above is directly conflicting with https://github.com/nodejs/nodejs.org/commit/31a50a692ee118350d3bc89193cdf2a3f7692539 - as if we have dueling format output

image

i re-generated my dependencies atop the lockfile - can you confirm you are using the latest too?

**Comment by ovflowd** [comment]
I mean yea, we updated the tailwind prettier plugin, so main branch has new rules 👀

**Comment by bmuenzenmeyer** [comment]
cursed pr
